src/exponential.jl

```diff
@@ -97,7 +97,35 @@ function expm_overapproximation(A::IntervalMatrix{T, Interval{T}}, t, p) where {
     return res
 end
 
+# Implementation of Prop. 1 in Althoff, Matthias, Bruce H. Krogh, and Olaf Stursberg.
+# "Analyzing reachability of linear dynamic systems with parametric uncertainties."
+# Modeling, Design, and Simulation of Systems with Uncertainties. Springer, Berlin, Heidelberg, 2011. 69-94.
 function _expm_remainder(A::IntervalMatrix{T}, t, p; n=checksquare(A)) where {T}
+    C = max.(abs.(inf(A)), abs.(sup(A)))
+    # compute Q = I + Ct + (Ct)^2/2! + ... + (Ct)^p/p!
+    Q = Matrix(Diagonal(ones(T, n)))
+
+    tⁱ = 1
+    i! = 1
+    Cⁱ = copy(C)
+    for i in 1:p
+        i! *= i
+        tⁱ *= t
+        Q = Q + Cⁱ * tⁱ/i!
+        Cⁱ = Cⁱ * C
+    end
+    M = exp(C*t)
+    Y  = M - Q
+    Γ = IntervalMatrix(fill(zero(T)±one(T), (n, n)))
+    E = Γ * Y
+    return IntervalMatrix(E) # See #73
+end
+
+# Estimates the sum of the series in the matrix exponential. See Theorem 1
+# in [1] Althoff, Matthias, Olaf Stursberg, and Martin Buss.
```

Comment:
```suggestion
# in [1] Matthias Althoff, Olaf Stursberg, and Martin Buss.
```

Reply:
I copied the reference from google scholar. There are different citation styles which one can choose, and I most frequently use [MLA](https://www.library.cornell.edu/sites/default/files/mla_style_revised.pdf), which happens to begin with the the last name of the first author and then use surname-last name of the other authors.

I'm fine to change to another standard style (APA, ISO690, MLA), i have no particular preference.

Reply:
Okay, we can keep it.